hail_scripts/reference_data/config.py

```diff
@@ -433,7 +450,11 @@ def custom_mpc_select(ht):
             },
         },
     },
+    'high_constraint_region_mito': {
+        '38': {
+            'version': 'Feb-15-2022',
+            'source_path': 'gs://seqr-reference-data/GRCh38/mitochondrial/Helix high constraint intervals Feb-15-2022.tsv',
+            'custom_import': import_locus_intervals,
+        },
+    },
 }
-
-CONFIG['dbnsfp_mito'] = {'38': deepcopy(CONFIG['dbnsfp']['38'])}
-CONFIG['dbnsfp_mito']['38']['filter'] = lambda ht: ht.locus.contig == 'chrM'
```

Comment:
I pushed the "is this mito" logic into the `filter` function itself, which I think is clearer than having a mutated config.